src/Admin/AbstractAdmin.php

```diff
@@ -656,6 +662,9 @@ public function getExportFields()
         return $fields;
     }
 
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
We recently removed all the inheritdoc we found.

This does not add any useful information, this is the default behavior of phpdoc